### PR TITLE
python-sympy and some packages it depends upon.

### PR DESCRIPTION
**Pull request by JPeterMugaas**

### CODE REVIEW

File: mingw-w64-python-mpmath/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#This value is here because some python package names have capital letters
+#while there is a strong preference for lowercase letters in ArchLinux and
+#MSYS packages.
+
+_realname=mpmath
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-${_realname}"  "${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+pkgver=1.0.0
+pkgrel=1
+pkgdesc="Some package (mingw-w64)"
```
Comment (Alexpux) [inline mingw-w64-python-mpmath/PKGBUILD]: Fix description

File: mingw-w64-python-sympy/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# Maintainer: Some One <some.one@some.email.com>
```
Comment (Alexpux) [inline mingw-w64-python-sympy/PKGBUILD]: Fix maintainer

File: mingw-w64-python-mpmath/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,112 @@
+# Maintainer: Some One <some.one@some.email.com>
```
Comment (Alexpux) [inline mingw-w64-python-mpmath/PKGBUILD]: Fix maintainer

File: mingw-w64-python-sympy/sympy2-i686.install [hunk 4]
```diff
@@ -0,0 +1,14 @@
+post_install() {
+  cd mingw32
+  local _prefix=$(pwd -W)
+  cd -
+  local _it
+  for _it in wheel2; do
```
Comment (Alexpux) [inline mingw-w64-python-sympy/sympy2-i686.install]: Does it really need to be wheel2?

File: mingw-w64-python-sympy/sympy2-x86_64.install [hunk 5]
```diff
@@ -0,0 +1,14 @@
+post_install() {
+  cd mingw64
+  local _prefix=$(pwd -W)
+  cd -
+  local _it
+  for _it in wheel2; do
```
Comment (Alexpux) [inline mingw-w64-python-sympy/sympy2-x86_64.install]: The same as previous

File: mingw-w64-python-sympy/sympy3-x86_64.install [hunk 6]
```diff
@@ -0,0 +1,14 @@
+post_install() {
+  cd mingw64
+  local _prefix=$(pwd -W)
+  cd -
+  local _it
+  for _it in wheel; do
```
Comment (Alexpux) [inline mingw-w64-python-sympy/sympy3-x86_64.install]: Here too

File: mingw-w64-python-sympy/sympy3-i686.install [hunk 7]
```diff
@@ -0,0 +1,14 @@
+post_install() {
+  cd mingw32
+  local _prefix=$(pwd -W)
+  cd -
+  local _it
+  for _it in wheel; do
```
Comment (Alexpux) [inline mingw-w64-python-sympy/sympy3-i686.install]: And here

**Comment by Alexpux** [comment]
@JPeterMugaas fix gmpy2 package as it not building

**Comment by JPeterMugaas** [comment]
Alright, I think I may have a fix for that in https://github.com/Alexpux/MINGW-packages/pull/4262 .  The issue is that there are symbolic links in the tarball and bsdtar will not work on those.  Ideally, software distributions should NOT have symbolic links at all (just several versions of the same file).  But barring that, bsdtar should solve the symbolic links and make a copy of the file in the appropriate place in Windows.

**Pull request merged**